source/api.rst

```diff
@@ -0,0 +1,71 @@
+.. toctree::
+
+API
+---
+
+It's possible to do all operations using the Onedata API.
+
+The official documentation is: https://onedata.org/#/home/api.
+
+.. important:: In order to be able to access the `Onedata APIs`, it's required
+    to generate an access token.
+
+Getting an API access token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tokens have to be generated from the `DataHub` (Onezone) interface as
+documented in :ref:`auth-token-using-web-interface` or using a command line
+call as documented hereafter.
+
+The following variables can be exported:
+
```

Comment:
It's good that the explanation of these variables is given. This makes it comprehensible to the reader. However, I suggest a table to make even clearer, and an example, so that the description is explicit.

Variable Name | Description | Example
-------------------|----------------|------------